apps/alerting/notifications/kinds/receiver.cue

```diff
@@ -11,12 +11,12 @@ receiver: {
 	}
 	pluralName: "Receivers"
 	current:    "v0alpha1"
-	codegen: {
-		ts: {enabled: false}
-		go: {enabled: true}
-	}
 	versions: {
 		"v0alpha1": {
+			codegen: {
+				ts: {enabled: false}
```

Comment:
@IfSentient typescript is still generated with this

Reply:
This should be fixed in the latest `apiserver-custom-routes`